app/services/make_an_offer.rb

```diff
@@ -12,12 +12,14 @@ class MakeAnOffer
   validate :validate_further_conditions
 
   def initialize(
+    auth:,
```

Comment:
How about passing in an `actor` into this class? Inside the `MakeAnOffer` constructor we can still instantiate the `Authorisation` object.

- This will hide the concept of authorisation into the service
- It will make it easier to instantiate the class
- It would make it easier to pass extra arguments to Authorisation if we need it

Reply:
Ok